src/UI/EditorGroupView.re

```diff
@@ -228,6 +228,32 @@ let make = (~state: State.t, ~windowId: int, ~editorGroup: EditorGroup.t, ()) =>
             shouldHighlightActiveIndentGuides
           />;
         | BufferRenderer.Welcome => <WelcomeView state />
+        | BufferRenderer.Terminal({id}) =>
+          let maybeFont =
```

Comment:
I'll refactor this to a separate component, and clean up the font handling, before bringing this in.